clientconn.go

```diff
@@ -137,7 +151,16 @@ func (cc *ClientConn) resetTransport(closeTransport bool) error {
 		if err != nil {
 			// TODO(zhaoq): Record the error with glog.V.
 			closeTransport = false
-			time.Sleep(backoff(retries))
+			if neterr, ok := err.(net.Error); ok {
```

Comment:
transport.ConnectionError is defined at https://github.com/grpc/grpc-go/blob/master/transport/transport.go#L398.